pkgs/by-name/s-/s-search/package.nix

```
@@ -0,0 +1,50 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  testers,
+  nix-update-script,
+}:
+
+buildGoModule (finalAttrs: {
+  pname = "s-search";
+  version = "0.7.2";
+
+  src = fetchFromGitHub {
+    owner = "zquestz";
+    repo = "s";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-5hkorROs11nrDK5/BBEPIugVYeVUWtAnpCBBuKTj15g=";
+  };
+
+  vendorHash = "sha256-0E/9fONanSxb2Tv5wKIpf1J/A6Hdge23xy3r6pFyV9E=";
+
+  ldflags = [
+    "-s"
+    "-w"
+  ];
+
```

```suggestion

  nativeBuildInputs = [ installShellFiles ];

  postInstall = lib.optionalString (stdenv.buildPlatform.canExecute stdenv.hostPlatform) ''
    installShellCompletion --cmd s \
      --bash <($out/bin/s completion bash) \
      --fish <($out/bin/s completion fish) \
      --zsh <($out/bin/s completion zsh)
  '';

```

https://github.com/zquestz/s/blob/7259a0cf06c719e72a3dbd29f01c9428d4a503a7/cmd/search.go#L64

---

good catch